src/containers/MainApp/index.jsx

```diff
@@ -15,10 +15,18 @@ import EditStaffer from '../EditStaffer';
 import CreateCollaborator from '../CreateCollaborator';
 import EditCollaborator from '../EditCollaborator';
 import EditCourse from '../EditCourse';
+import SitewideBanner from '../../components/SitewideBanner';
 
 const MainApp = () => (
   <div>
     <Header />
+    <SitewideBanner
+      message={process.env.SITEWIDE_BANNER_CONTENT || ''}
+      type="primary"
+      dismissible
+      cookieName="sitewidebannerDismissed"
```

Comment:
nit: please make the cookie name unique enough, such as publisherSiteWideBannerViewed or publisherSiteWideBannerDismissed